### DISCUSSION TITLE
UBA in Python 3.10.1 with pySide6.2 and pyqt6.2

### DISCUSSION THREAD

**Discussion by dmccon** [eliranwong/UniqueBible#856] [Q&A] [answered]
How do I run this? do I need to downgrade Python? Is there a plan for a new uba.py? Thanks for any help I can get. I'm basically zero-knowledge.

**Comment by otseng** [comment]
Currently UBA does not support 3.10.  Hopefully sometime soon will be able to modify UBA to support it.  In the meantime, will need to run on 3.9.  

If anybody can fix this and submit a pull request, that'd be appreciated.

**Comment by dmccon** [comment, marked as answer]
Thank you!
I kind of anticipated that as I tried to research it, and ended up getting it to run on 3.8.8. Have a great New Year in Christ!

Sent from my iPhone

On Dec 29, 2021, at 08:05, Oliver Tseng ***@***.***> wrote:

﻿

Currently UBA does not support 3.10. Hopefully sometime soon will be able to modify UBA to support it. In the meantime, will need to run on 3.9.

If anybody can fix this and submit a pull request, that'd be appreciated.

—
Reply to this email directly, view it on GitHub<https://github.com/eliranwong/UniqueBible/discussions/856#discussioncomment-1883367>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/APAVOKK4AUQ6HEDI2Q3BAYDUTMIQ5ANCNFSM5K24KOQQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Comment by eliranwong** [comment]
I am running UBA with python 3.10.4 on 2 devices.
